src/main.rs

```diff
@@ -1,10 +1,39 @@
+extern crate egg_mode;
+extern crate dotenv;
 extern crate fs2;
 extern crate rand;
+extern crate tokio_core;
 
 mod name_gen;
 mod epic;
 
+use dotenv::dotenv;
+use std::env;
+use name_gen::*;
+use egg_mode::tweet::DraftTweet;
+use tokio_core::reactor::Core;
 
 fn main() {
-    println!("{}", name_gen::random_msg());
-}
+    dotenv().ok();
+
+    let consumer_key = env::var("CONSUMER_KEY").unwrap();
+    let consumer_secret = env::var("CONSUMER_SECRET").unwrap();
+    let access_token = env::var("ACCESS_TOKEN").unwrap();
+    let access_token_secret = env::var("ACCESS_TOKEN_SECRET").unwrap();
+
+    let con_token = egg_mode::KeyPair::new(consumer_key, consumer_secret);
+    let access_token = egg_mode::KeyPair::new(access_token, access_token_secret);
+    let token = egg_mode::Token::Access {
+        consumer: con_token,
+        access: access_token,
+    };
+
+    let epic = name_gen::random_msg();
+    let tweet = DraftTweet::new(&epic);
+    
+    println!("{:?}", tweet);
```

Comment:
What does the output of this look like? Do we just want to print the message, or something else pretty?

Reply:
This was for debugging purposes (I was curious to what DraftTweet looks like internally) and I forgot to remove it. I'll remove it now


The output looks like this:

`DraftTweet { text: "Super Mario Iliad\n\n(https://en.wikipedia.org/wiki/Iliad)", in_reply_to: None, auto_populate_reply_metadata: None, exclude_reply_user_ids: None, attachment_url: None, coordinates: None, display_coordinates: None, place_id: None }`